lib/xlsx/xform/core/core-xform.js

```diff
@@ -27,6 +27,7 @@ var CoreXform = module.exports = function() {
     'cp:revision': new DateXform({tag: 'cp:revision'}),
     'cp:version': new StringXform({tag: 'cp:version'}),
     'cp:contentStatus': new StringXform({tag: 'cp:contentStatus'}),
+    'cp:contentType': new DateXform({tag: 'cp:contentType'}),
```

Comment:
I would think this should be a `StringXform`?

Reply:
Ah, you're right, nice catch. I had that originally, but started playing around because that enfant terrible spreadsheet still didn't parse :)

Fixed.